webpack/assets/javascripts/react_app/components/HostDetails/DetailsCard/index.js

```diff
@@ -0,0 +1,141 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import {
+  DescriptionList,
+  DescriptionListTerm,
+  DescriptionListGroup,
+  DescriptionListDescription,
+  Card,
+  CardHeader,
+  CardTitle,
+  CardBody,
+  CardActions,
+  ClipboardCopy,
+  Divider,
+} from '@patternfly/react-core';
+import { UserIcon } from '@patternfly/react-icons';
+import { foremanUrl } from '../../../common/helpers';
+import PowerStatusDropdown from './PowerStatusDropdown';
+import { translate as __ } from '../../../common/I18n';
+import SkeletonLoader from '../../common/SkeletonLoader';
+import { STATUS } from '../../../constants';
+
+const DetailsCard = ({
+  id: hostID,
+  ip,
+  ip6,
+  mac,
+  comment,
+  owner_id: ownerID,
+  owner_name: ownerName,
+  hostgroup_name: hostgroupName,
+  hostgroup_id: hostgroupID,
+  permissions: { power_hosts: canUsePower },
+  status,
+}) => (
+  <Card isHoverable>
+    <CardHeader>
+      <CardTitle>{__('Details')}</CardTitle>
+      <CardActions>
+        <PowerStatusDropdown hostID={hostID} hasPowerPermission={canUsePower} />
+      </CardActions>
+    </CardHeader>
+    <CardBody>
+      <DescriptionList
+        isAutoColumnWidths
+        columnModifier={{
+          default: '2Col',
+        }}
+      >
+        <DescriptionListGroup>
+          <DescriptionListTerm>
+            {ip6 ? __('IP6 Address') : __('IP Address')}
+          </DescriptionListTerm>
+          <DescriptionListDescription>
+            <SkeletonLoader status={status}>
+              <ClipboardCopy variant="inline-compact">
+                {ip6 || ip}
+              </ClipboardCopy>
```

Comment:
Would this show the empty state for the loader?

Reply:
no, good catch